tools/pipelines/build-client.yml

```diff
@@ -139,3 +139,25 @@ extends:
     - syncpack:versions
     - check:versions
     - prettier
+    additionalSteps:
```

Comment:
For this bit:
- `workingDir` needs to go one folder further down, right? `./packages/tools/devtools` is not a "buildable" folder.
- Let's use the same `workingDir` for the creation of the `.env` file. Right now it gets created at the root of the repo, so I don't think the next step (running from a different working dir) will use it.
- Update the `displayName` of the third task to clarify what is the artifact being published.

Reply:
I'm also thinking that maybe running `npm run build` maybe will do nothing because the package was built earlier in the pipeline, and nothing has changed that should require a rebuild, so the command might do nothing. Try it, and if that's the case, we might want to run `npm run webpack` directly. Which btw, does work if we just provide the environment variable, it doesn't need an `.env` file. It seems like fluid-build (called by `npm run build`) is the one that isn't passing the env variable correctly to the many tasks it potentially executes. So if we end up having to call `npm run webpack` directly, we can skip creating the `.env` file and use the `env:` section of an ADO task to provide the value.

Reply:
Make two additional steps then we won't need to build devtools again. Thanks for fixing the env vars issue. That is awesome!

Reply:
Just note that the env vars issue fix might only be effective next time we bump the build-tools dependency in the client release group, so more reason to not depend on it right now.

Reply:
Gotcha